packages/collector/test/tracing/cloud/aws-sdk/v2/test.js

```diff
@@ -16,7 +16,8 @@ const globalAgent = require('../../../../globalAgent');
 const { verifyHttpRootEntry, verifyExitSpan } = require('@instana/core/test/test_util/common_verifications');
 const { promisifyNonSequentialCases } = require('./promisify_non_sequential');
 
-const functionName = 'team-nodejs-invoke-function';
+// We are reusing the same function used for testing Lambda functions, as we are simply invoking it here.
+const functionName = 'nodejs-tracer-lambda';
```

Comment:
Reusing the same Lambda function throughout the tests to simplify maintenance by avoiding multiple functions, and so far, it's working as expected.

Reply:
suggestion: nodejs-tracer-lambda-v2' & nodejs-tracer-lambda-v3';

Reply:
why we need 2 lambda functions?

Reply:
If I run a test locally and they run on CI in parallel, it will clash or? Thats why the whole aws tests were designed to create unique fn names

Reply:
In our test, we are primarily invoking the function and retrieving its details using `GetFunction`. Lambda is capable of handling concurrent requests for both invocation and fetching function information via `GetFunction`. I don't find any other usecase in our test.

Reply:
If its only for invoking, then yes thats okay. I thought this functionName is also used for creating functions.

Reply:
No, we are not creating the functions during the test, we use existing function, that's why I used one function for all lambda related tests.

Reply:
Okay. Did u manually create the functions on 7x?

Reply:
yes

Reply:
Could you pls expand your comment with this knowledge? Thanks <2